.github/config.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Disable label by Drift product
+product: false
+language:
+  # Disable issue labeling (not accurate)
+  issue: false
+  # Enable pr labeling
+  pullrequest: true
+  labelprefix: 'lang: '
+  # To re-map extension to languages
+  extensions:
+    JSON: ['JSON']
+    typescript: ['ts']
+  # To map languages based on project directory structure:
+  paths:
+    src:
+      frontend: 'go'
+      cartservice: 'c#'
```

Comment:
it's c# indeed but in drift systems it's labeled as "dotnet", so we should use what DRIFT "understands".